### ISSUE TITLE
About the tested IXI images

### ISSUE THREAD

**Issue by PWonjung** [BioMedAI-UCSC/InverseSR#11]
Hello, author of InverseSR.
We are trying to utilize your nice work as our baseline in our paper for SR.
Due to the large resource requirement, we just want to use the same test images and then compare the evaluated PSNR, .. and other factors written in your paper. 
I wonder the test images are the list below ( I saw it on your github code).
Thank you so much for your nice work. 

![image](https://github.com/BioMedAI-UCSC/InverseSR/assets/16303328/24b74360-2613-496c-bff8-8b77393a5799)

**Comment by jueqiw** [comment]
Hi,

Thanks for your interest in our work! Yes, these are 100 subjects used for testing. 

Please don't hesitate to let me know if you have future questions!

Jueqi

**Comment by 06Liz** [comment]
Hello, may I ask why the reconstructed syns images is incomplete and half missing when I test T1 images in IXI dataset?

**Comment by jueqiw** [comment]
Can you share with me what test T1 image and code you use?

**Comment by 06Liz** [comment]
The dataset dataset T1 022：http://brain-development.org/ixi-dataset/
![图片1](https://github.com/BioMedAI-UCSC/InverseSR/assets/130451549/0d73ae08-a437-43bf-a144-0171a1cc02ca)
![图片2](https://github.com/BioMedAI-UCSC/InverseSR/assets/130451549/3e533a43-5b8b-4df7-935e-660a5b52f8b0)
one is syns,the others is target.
The code I'm running is /job_script/InverseSR(ddim).sh

**Comment by jueqiw** [comment]
Hi, the image put into the model should be registered into a MNI space (the brain needs to be roughly in the center of the image), following what [the pretrained Brain LDM](https://arxiv.org/pdf/2209.07162) has done, we use UniRes to perform that.

**Comment by PWonjung** [comment]
Hello I have one more question.
I tried to use BRGM_decoder, but I found that the top of the skull is prone to blurred. 
Is it inherited shortcoming of the inverseSR due to using corrupted image, or do you think there is any possible reason for this? 
Thanks for you attention.
![image](https://github.com/BioMedAI-UCSC/InverseSR/assets/16303328/b41c505a-5b07-402d-a1e1-92ce3f742bae)

**Comment by jueqiw** [comment]
Hi, I do think it is because the pre-trained model used corped images (it said in the [Brain LDM](https://arxiv.org/pdf/2209.07162) paper, they cropped the image to obtain a volume of 160 × 224 × 160) for training. So for some images, the top of the skull is cropped, making the generated skull blurry.
 
Thanks,

**Comment by 06Liz** [comment]
> Hi, the image put into the model should be registered into a MNI space (the brain needs to be roughly in the center of the image), following what [the pretrained Brain LDM](https://arxiv.org/pdf/2209.07162) has done, we use UniRes to perform that.
Hi,I tested the image diretly using nii format,and the pre-dim was axial.The result is the same as before,only half and generation is fuzzy,syns is not clear as the target.

**Comment by 06Liz** [comment]
Hello,cloud you help me slove the problem?

> 数据集 T1 022：http://brain-development.org/ixi-dataset/ ![图片1](https://private-user-images.githubusercontent.com/130451549/344399715-0d73ae08-a437-43bf-a144-0171a1cc02ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5OTI5NTUsIm5iZiI6MTcxOTk5MjY1NSwicGF0aCI6Ii8xMzA0NTE1NDkvMzQ0Mzk5NzE1LTBkNzNhZTA4LWE0MzctNDNiZi1hMTQ0LTAxNzFhMWNjMDJjYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwM1QwNzQ0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZTU0ZmI2OTI2YWRlNzQ5NGZlODNhMjAxYWE4YWU4ZTFlNTZiODNlYzdjNjFjNDVlOGZkN2ZhZTIwOWZlZGYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BWcjkLQxjRiNCY1Sc_4a-s50MRjTd2ckEK3cGytvHtk) ![图片2](https://private-user-images.githubusercontent.com/130451549/344399733-3e533a43-5b8b-4df7-935e-660a5b52f8b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5OTI5NTUsIm5iZiI6MTcxOTk5MjY1NSwicGF0aCI6Ii8xMzA0NTE1NDkvMzQ0Mzk5NzMzLTNlNTMzYTQzLTViOGItNGRmNy05MzVlLTY2MGE1YjUyZjhiMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwM1QwNzQ0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YzY3MDFkODk5ZmQyYzA5M2I1NWJmM2Q2ODIxMjA2MTg2MTY3N2I5MzFhNzZlY2U2Yjg4NTI3MTRlNDIzMDM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ql0eINW4rH8kQTlfFBi7qynCQOxD61F_wz3m3BryzkY) 一个是 syns，另一个是 target。我正在运行的代码是 /job_script/InverseSR（ddim）.sh

@wendy-xiaozong @06Liz

**Comment by 06Liz** [comment]
Can you give us the details to slove the problem of only half of images? @wendy-xiaozong

**Comment by jueqiw** [comment]
Hi @06Liz ,

I uploaded a new file: ur_IXI022-Guys-0701-T1.nii.gz to the google drive link: https://drive.google.com/drive/folders/110l68um6gUJzECIv0AyF-4Fcw0rrQgA9

I think you can get the whole image from it. The half image problem to me is still that the brain is not in the center and should go through the UniRes before being put into the model.

**Comment by 06Liz** [comment]
Why is the voxel 1X1X1 before and after reconstruction? Where is the super-resolution reconstruction represented? Where is the 3D reconstruction represented?Because the output is the two-dimensional.

**Comment by 06Liz** [comment]
How to see the input is 3D how many mm to reconstruction?Target is 1mm，so where are the 4mm
and 8mm inputs respresents? Where is the 3D reconstruction represented?Because the output is the two-dimensional. @wendy-xiaozong
